src/layout/Group/RepeatingGroupContainer.tsx

```diff
@@ -42,6 +42,7 @@ export function RepeatingGroupContainer({ containerDivRef }: IGroupProps): JSX.E
   const { textResourceBindings, id, edit, type } = node.item;
 
   const numRows = visibleRowIndexes.length;
+  const firstIndex = visibleRowIndexes[0];
```

Comment:
Is the lint-warning here correct? If this is unused, maybe we could remove it all together

Reply:
Strange. From SonarCloud? I don't always even read those warnings, they're often funky. It's being used in a `useEffect()` below, so absolutely needed.

Reply:
Sorry! I should have specified that i meant the entire function. Github does not allow me to place the comment anywhere without a diff :sweat_smile: 
![image](https://github.com/Altinn/app-frontend-react/assets/26043795/bbc667ad-c214-4a79-a0be-43a40ed70bb5)

Reply:
Ah, yes, I don't know - but I would think that's something @bjosttveit is removing if he doesn't need it in the validation rewrite.